python/core/auto_generated/layout/qgslayoutitemmap.sip.in

```diff
@@ -508,12 +508,15 @@ be used instead.
 
     void setLayers( const QList<QgsMapLayer *> &layers );
 %Docstring
-Sets the stored ``layers`` set. If empty, the current project layers will
-be used instead.
+Sets the stored ``layers`` set. If empty, the current project layers or the
+layer specified in the map theme (if the follow visibility preset flag is set)
```

Comment:
This change isn't correct, is it?

Reply:
Here's the implementation: https://github.com/qgis/QGIS/blob/master/src/core/layout/qgslayoutitemmap.cpp#L2130

I just wanted to point it out what are the options here. How would you suggest to improve this text?

Reply:
@nyalldawson gentle ping: if this is a blocker I can remove the change.

Reply:
Sorry, slipped through the cracks!

"If empty, the current project layers or thelayer specified in the map theme"

This is incorrect -- it should be something like "If the map theme is set then this list will be ignored". The map theme overrides anything specified in a setLayers call.